src/NServiceBus.Persistence.TestRunner/NServiceBus.Persistence.TestRunner.csproj

```diff
@@ -1,107 +1,27 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
-    <ProjectGuid>{2A4EE72F-E17F-4858-ADAA-34FB47068F5C}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>NServiceBus.Persistence.TestRunner</RootNamespace>
+    <PlatformTarget>x64</PlatformTarget>    
+    <TargetFramework>net452</TargetFramework>
     <AssemblyName>NServiceBus.Persistence.TestRunner</AssemblyName>
```

Comment:
AssemblyName is by default inferred from the project file name. so this is redundant